index.html

```diff
@@ -51,23 +52,28 @@ <h1>Im Lucca
     <section class="all1">
         <div class="document1">
             <img class="one" src="images/SnapshootPortfolio(2).jpg" alt="">
+            <img class="one11" src="images/SnapshootPortfolio.jpg" alt="">
             <h2 class="tonic">Tonic</h2>
-        </div>
-        <div class="part1">
-            <div class="list">
+        
+            <div class="part1">
                 <ul class="inline">
                     <li><img class="words" src="images/Frame2.jpg" alt=""></li>
                 </ul>
             </div>
           
             <p class="subtitle" >A daily selection is privaly 
```

Comment:
- [ ] This taking so much space in your page, kindly update it because I think you mistakenly changed its css
![image](https://user-images.githubusercontent.com/62705947/187305806-80007334-4492-4c71-9c2b-38615a9329c4.png)